myhpom/templates/myhpom/dashboard.html

```diff
@@ -66,7 +66,7 @@ <h2 class="h6 dashboard__section-title">
                             <span class="h5 dashboard-profile-information__status">I am not an organ donor.</span>
                         </div>
                         <div>
-                            <a href="http://donatelifenc.org/Register" alt="Donate Life" class="dashboard-profile-information__organ-donor-link">
+                            <a href="http://donatelifenc.org/Register" alt="Donate Life" class="dashboard-profile-information__organ-donor-link" target="_blank">
```

Comment:
When specifying _blank, also include `rel="noopener noreferrer"` to avoid exploits via window.opener.

Reply:
+1